linkerd/proxy/http/src/balance.rs

```diff
@@ -1,59 +1,9 @@
-use crate::Error;
-use hyper::body::HttpBody;
-pub use hyper_balance::{PendingUntilFirstData, PendingUntilFirstDataBody};
-use rand::thread_rng;
-use std::{hash::Hash, marker::PhantomData, time::Duration};
-use tower::discover::Discover;
-pub use tower::{
-    balance::p2c::Balance,
-    load::{Load, PeakEwmaDiscover},
-};
+use hyper_balance::{PendingUntilFirstData, PendingUntilFirstDataBody};
+use linkerd_proxy_balance as balance;
 
-/// Configures a stack to resolve `T` typed targets to balance requests over
-/// `M`-typed endpoint stacks.
-#[derive(Debug)]
-pub struct Layer<A, B> {
-    decay: Duration,
-    default_rtt: Duration,
-    _marker: PhantomData<fn(A) -> B>,
-}
+pub type Body<B> = PendingUntilFirstDataBody<balance::Handle, B>;
 
-// === impl Layer ===
+pub type EwmaConfig = balance::EwmaConfig;
```

Comment:
what's up with the type alias? is there a plan for these types to differ based on protocol?

Reply:
eh, it's basically just a `pub use` for the moment so callers don't need to know about the balance crate explicitly.